src/loaders/jpg/tvgJpgd.cpp

```diff
@@ -667,7 +667,7 @@ inline void jpeg_decoder::stuff_char(uint8_t q)
 // Retrieves one character from the input stream, but does not read past markers. Will continue to return 0xFF when a marker is encountered.
 inline uint8_t jpeg_decoder::get_octet()
 {
-    bool padding_flag;
+    bool padding_flag = false;
```

Comment:
I think there are 2 options:
1. just to shutdown, TVG_UNUSED may work for this
2. It's better to fix inside get_char().   see: 668 line.

Reply:
> 2\. It's better to fix inside get_char().   see: 668 line.

Fixed, thanks!